### PR TITLE
Ts tests README

### LINKED ISSUE #1872: README content needs to be updated for ts-tests.
### Context
Now that there is more and more content/files in `ts-tests`, the current README content for ts-tests is insufficient. It is inconvenient for others to use. We need to update with new content.

### PR DESCRIPTION
**Pull request by 0xverin**
### Context

resolves: #1872

### CODE REVIEW

**Comment by 0xverin** [comment]
Please feel free to point out any additional content that needs to be added. cc @grumpygreenguy

File: tee-worker/ts-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+`cd tee-worker/ts-tests`
+
+`yarn`
+
+## Type Generated
+
+Update parachain metadata: `yarn workspace parachain-api update-metadata`
+
+Update parachain metadata: `yarn workspace sidechain-api update-metadata`
```
Comment (kziemianek) [inline tee-worker/ts-tests/README.md]: This command is for sidechain but comments suggests parachain.

File: tee-worker/ts-tests/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+## Description
+
+ts-tests of tee-worker
+
+## Environment
+
+node>=16, yarn v3
```
Comment (kziemianek) [inline tee-worker/ts-tests/README.md]: It's confirmed that it doesn't work with node 20 :(

File: tee-worker/ts-tests/README.md (hunk 1 above)
Comment (0xverin) [reply tee-worker/ts-tests/README.md]: fixed!

File: tee-worker/ts-tests/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Generate parachain type: `yarn workspace parachain-api build`
+
+Generate sidechain type: `yarn workspace sidechain-api build`
+
+## Local
+
+[Start parachain && tee-worker](https://github.com/litentry/litentry-parachain/blob/dev/README.md)
+
+`echo -e "NODE_ENV=local\nWORKER_END_POINT=wss://localhost:2000\nSUBSTRATE_END_POINT=ws://localhost:9944\nID_HUB_URL=http://localhost:3000" > ./integration-tests/.env.local`
```
Comment (Kailai-Wang) [inline tee-worker/ts-tests/README.md]: Isn't this in `./integration-tests/.env.local` already?

File: tee-worker/ts-tests/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## Usage
```
Comment (Kailai-Wang) [inline tee-worker/ts-tests/README.md:29]: Can you add commands for DI-sample code execution? We have two variants.

File: tee-worker/ts-tests/README.md (hunk 3 above)
Comment (0xverin) [reply tee-worker/ts-tests/README.md]: I was mistaken; I thought this required manual configuration.

File: tee-worker/ts-tests/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+## Installation
+
+`cd tee-worker/ts-tests`
+
+`yarn`
+
+## Type Generated
+
+Update parachain metadata: `yarn workspace parachain-api update-metadata`
```
Comment (arieljbirnbaum) [inline tee-worker/ts-tests/README.md]: Should note that this requires a working runner on localhost:9944

**Review by arieljbirnbaum — CHANGES_REQUESTED** [review]
one small but important change; LGTM otherwise :+1:

File: tee-worker/ts-tests/README.md (hunk 5 above)
Comment (Kailai-Wang) [reply tee-worker/ts-tests/README.md]: Following up on this: 

generating sidechain metadata requires that worker is running, doesn't it

**Review by arieljbirnbaum — APPROVED** [review]

**Pull request merged**